### PR TITLE
edits to ITP timeline based on feedback

### PR DESCRIPTION
**Pull request by SallyMcGrath**
data flows was misaligned, and what about christmas

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
oh daniels' away!

File: org-cyf-itp/content/guides/teaching/timeline/index.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ emoji= "📅"
...
 
-The course starts on the 1st of the month but there may be 7 days until the first Saturday, so we have written this to cover that variance.
+The course starts on the 1st of the month but there may be 7 days until the first Saturday, so we have written this to follow that timeline. 
 
-There are 12 learning weeks in the curriculum, which you will schedule over a maximum of 16 weeks. Most people should complete this course in 12 weeks of classes. Some will go faster (and this is encouraged). A few people will have adverse life events and need a couple of weeks to catch up. Once a learner is a full module behind, it's not realistic for them to catch up.
+There are 12 learning weeks in the curriculum, which you will schedule over a maximum of 16 weeks. This padding will help you account for things like Christmas or Eid or other movable feasts. The course allows for variance in ability and previous experience. Most people should complete this course in 12 weeks of classes. Some will go faster (and this is encouraged). A few people will have adverse life events and need a couple of weeks to catch up. Once a learner is a full module behind, it's not realistic for them to catch up.
```
Comment (SallyMcGrath) [inline org-cyf-itp/content/guides/teaching/timeline/index.md:13]:
```suggestion
There are 12 learning weeks in the curriculum, which you will schedule over a maximum of 16 weeks. This padding will help you account for things like Christmas, or Eid and other movable feasts. The course allows for variance in ability and previous experience. Most people should complete this course in 12 weeks of classes. Some will go faster (and this is encouraged). A few people will have adverse life events and need a couple of weeks to catch up. Once a learner is a full module behind, it's not realistic for them to catch up.
```

**Review by SallyMcGrath — COMMENTED** [review]
self reviewing cause i'm the only one heeeeeeeeere

**Comment by illicitonion** [comment]
LGTM! I'm impressed that you actually had comments in your self-review 😂

**Pull request merged**